File: Ninja Slayground 2.0/2. Nth Fibonacci Number/Solution.java
// Learn the code, And copy and paste is not allowed. You are required to type...

// Follow this account, for all future code updates on GitHub.

import java.util.Scanner;
public class Solution {


	public static void main(String[] args) {
		
		/* Your class should be named Solution.
	 	* Read input as specified in the question.
	 	* Print output as specified in the question.
		*/
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int f = 0, s = 1;
		for(int i = 1; i < n; i++) {
			int t = s + f;
			f = s;
			s = t;
		}
		System.out.println(s);
		
	}

}

File: Ninja Slayground 2.0/2. Nth Fibonacci Number/Solution.py
#Learn the code, And copy and paste is not allowed. You are required to type...

#Follow this account, for all future code updates on GitHub.

from math import *
from collections import *
from sys import *
from os import *

## Read input as specified in the question.
## Print output as specified in the question.

n = int(input())
l = [0, 1]
for i in range(1, n):
    l.append(l[-1] + l[-2])
print(l[-1])